spec.bs

```diff
@@ -2137,6 +2139,35 @@ The output of running a Protected Audience `scoreAd()` script.
 
 </dl>
 
+This is returned via the following type:
+<pre class="idl">
+dictionary ScoreAdOutput {
+  required double desirability;
+  double bid;
+  boolean allowComponentAuction = false;
+};
+</pre>
+
+<div algorithm>
+
+To <dfn>process scoreAd output</dfn> given a |result|, a [=ECMAScript/Completion Record=]:
+  1. If |result| is an an [=abrupt completion=], return failure.
+  1. TODO: the code has special handling of where the return value is a straight up double, do we want to keep it? It is in the explainer.

```

Comment:
Is the behavior you're describing here what Chromium currently implements? Who is the right person on the PA team to make this call?

Reply:
yes, that's the current implementation.
https://source.chromium.org/chromium/chromium/src/+/refs/heads/main:content/services/auction_worklet/seller_worklet.cc;drc=b8a0323a84f483b25e94b3a24d80fda16c5dd1ae;l=982

Reply:
Got it. A web platform test for this would be great if possible. It's unclear how many if any of the WebIDL bindings behavior that we have to implement manually are tested here, but it would be great to get as much coverage as we can.